README.md

```diff
@@ -1,62 +1,82 @@
+
+## Building and Running
+
+1. Connect a USB cable between the USB port on the target and the host computer.
+2. Run the following command to build the example project, program the microcontroller flash memory, and open a serial terminal:
+    ```bash
+    $ mbed compile -m <TARGET> -t <TOOLCHAIN> --flash --sterm
+    ```
+(Note: Mbed CLI command-line option "--sterm" is used to open serial terminal after flashing.)
 
+The binary is located at `./BUILD/<TARGET>/<TOOLCHAIN>/mbed-os-example-cpu-stats.bin` 
+
+Alternatively, you can manually copy the binary to the board, which you mount on the host computer over USB.
+
+Depending on the target, you can build the example project with the `GCC_ARM`, `ARM` or `IAR` toolchain. After installing Arm Mbed CLI, run the command below to determine which toolchain supports your target:
+
+```bash
+$ mbed compile -S
 ```
-[snip]
-+------------------+-------+-------+------+
-| Module           | .text | .data | .bss |
-+------------------+-------+-------+------+
-| [lib]\c_w.l      | 11473 |    16 |  348 |
-| [lib]\cpprt_w.l  |    36 |     0 |    0 |
-| [lib]\fz_wm.l    |    18 |     0 |    0 |
-| [lib]\m_wm.l     |    48 |     0 |    0 |
-| anon$$obj.o      |    32 |     0 | 1024 |
-| main.o           |   530 |     8 |    0 |
-| mbed-os\drivers  |   165 |     0 |    0 |
-| mbed-os\events   |  2139 |     8 | 1568 |
-| mbed-os\features |   132 |     0 |  304 |
-| mbed-os\hal      |  1910 |    34 |  128 |
-| mbed-os\platform |  3695 |   104 |  604 |
-| mbed-os\rtos     | 17302 |  2310 | 4592 |
-| mbed-os\targets  | 10707 |   112 |  324 |
-| Subtotals        | 48187 |  2592 | 8892 |
-+------------------+-------+-------+------+
-Total Static RAM memory (data + bss): 11484 bytes
-Total Flash memory (text + data): 50779 bytes
-
-Image: .\BUILD\K64F\ARM\mbed-os-example-cpu-stats.bin
 
+## Expected output
+The LED blinks and the serial terminal shows an output similar to the following:
+```
+--- Terminal on /dev/ttyACM0 - 9600,8,N,1 ---
+Uptime: 2000061 Idle Time: 1999756 Sleep time: 0 DeepSleep time: 1999756 Idle: 99% Usage: 1%
+Uptime: 4000061 Idle Time: 3901336 Sleep time: 0 DeepSleep time: 3901336 Idle: 95% Usage: 5%
+Uptime: 6000061 Idle Time: 5803892 Sleep time: 0 DeepSleep time: 5803892 Idle: 95% Usage: 5%
+Uptime: 8000061 Idle Time: 7706632 Sleep time: 0 DeepSleep time: 7706632 Idle: 95% Usage: 5%
+Uptime: 10000061 Idle Time: 9609128 Sleep time: 0 DeepSleep time: 9609128 Idle: 95% Usage: 5%
+Uptime: 12000061 Idle Time: 11510770 Sleep time: 0 DeepSleep time: 11510770 Idle: 95% Usage: 5%
+Uptime: 14000061 Idle Time: 13410246 Sleep time: 0 DeepSleep time: 13410246 Idle: 94% Usage: 6%
+Uptime: 16000030 Idle Time: 14309689 Sleep time: 0 DeepSleep time: 14309689 Idle: 44% Usage: 56%
```

Comment:
Is it possible to align the fields? (check if this is supported in minimal printf)